ECMA2Yaml/UndocumentedApi/Validator.cs

```diff
@@ -40,7 +40,18 @@ public static ValidationResult ValidateReturnValue(ReflectionItem item)
             {
                 return ValidationResult.NA;
             }
-            return ValidateSimpleString(item.Docs.Returns, ReturnsLengthRequirement);
+
+            var result = ValidateSimpleString(item.Docs.Returns, ReturnsLengthRequirement);
+
+            // fix bug 84378(https://ceapex.visualstudio.com/web/wi.aspx?pcguid=7d644393-99ad-41c8-ac53-7fa79294c720&id=84378)
+            if ((result == ValidationResult.Missing) && (item.ItemType == ItemType.Field || item.ItemType == ItemType.Property))
```

Comment:
we need to confirm with Maira about the UnderDoc status here, my guess is that she wants UnderDoc to be NA too, which means we can just add Field and Property to the `if` above. I'll @ her in the bug.

Reply:
confirmed with Maira, we still need the underdoc status. So we are good to go.